Style-guide/README.md

```diff
@@ -81,39 +78,24 @@ Descriptive and consistent naming makes software easier to read and understand.
 - naming methods for their side effects
   - verb methods follow the -ed, -ing rule for the non-mutating version
   - noun methods follow the formX rule for the mutating version
-  - boolean types should read like assertions
+  - boolean types should read like assertions (`isLoading`, `isHidden`, etc)
   - protocols that describe _what something is_ should read as nouns
   - protocols that describe _a capability_ should end in _-able_ or _-ible_
 - using terms that don't surprise experts or confuse beginners
-- generally avoiding abbreviations
-- using precedent for names
+- generally avoiding abbreviations // a few well-known ones are allowed, such as, `Html` and `Id`. Any others?
+- using precedent for names // ???
 - preferring methods and properties to free functions
-- casing acronyms and initialisms uniformly up or down
+- casing acronyms and initialisms uniformly up or down // I think we use camelcase, as in `Id` and `Html`
```

Comment:
(I've had to change the line above to trigger a diff, else we couldn't comment on the line. The inline comment will be removed.)

I think we use camelcase, as in `Id` and `Html` instead.

Reply:
We use `ID`, unless it's a prefix `idOfSomething`.

Reply:
Right, so it's full upper in the case of a type and full lower in the case of part of an identifier.

Reply:
In iOS SDK acronyms are always uppercased in types , e.g. `HTTPURLResponse`. If there are places we're not doing it, we should fix it.